pkg/eventing/v1/client_mock.go

```diff
@@ -115,6 +115,10 @@ func (c *MockKnEventingClient) UpdateTrigger(ctx context.Context, trigger *event
 	return mock.ErrorOrNil(call.Result[0])
 }
 
+func (c *MockKnEventingClient) UpdateTriggerWithRetry(ctx context.Context, name string, updateFunc TriggerUpdateFunc, nrRetries int) error {
```

Comment:
Using the function in `client_mock_test.go` should be sufficient to get rid of codecov warning. Also for the other mock functions as well.